contrib/scripts/minikube.sh

```diff
@@ -1,22 +0,0 @@
-#!/bin/bash
-
-set -eux
-
-export MINIKUBE_NETWORK_PLUGIN="cni"
-export MINIKUBE_EXTRA_CONFIG="kubelet.network-plugin=cni"
-export MINIKUBE_MEMORY=5120
-unset CONTAINER_ENGINE
-
-minikube start
-# TODO(mrostecki): Support cri-o and buildah.
-eval $(minikube docker-env)
-
-make docker-images-all DOCKER_IMAGE_TAG=dev
-
-cp "install/kubernetes/quick-install.yaml" /tmp/cilium-minikube.yaml
-
-sed -i 's|latest|dev|g' /tmp/cilium-minikube.yaml
-sed -i 's|docker.io/||g' /tmp/cilium-minikube.yaml
-sed -i 's|imagePullPolicy: Always|imagePullPolicy: Never|g' /tmp/cilium-minikube.yaml
-
-kubectl create -f /tmp/cilium-minikube.yaml
```

Comment:
Instead of removing this script and the minikube target as there might be developers who depend on it (I'm not sure who though), maybe we can replace the highlighted lines with the cilium-cli?

Reply:
I've asked @mrostecki whether the script is still needed.

Reply:
I didn't use the script lately.

The question is if anyone finds it useful. If not, I'm fine with removing it. If yes, then I can follow up with switching to cilium-cli in it.